lib/index.d.ts

```diff
@@ -13,13 +15,8 @@ export type ServiceClientOptions = {
     operation: string;
     path?: string;
     pathParams?: {[key: string]: string};
-    payload?: 
-        | stream.Readable
-        | Buffer
-        | string
-        | Uint8Array
-        | {[key: string]: string | string[]};
-    queryParams?: {[key: string]: string | string[]};
+    payload?: string | Buffer | Stream.Readable | object;
```

Comment:
Instead of writing these out, please reference the Payload type seen here in the Wreck library: https://github.com/hapijs/wreck/blob/master/lib/index.d.ts#L291-L297

Reply:
This would become super hacky as the type `Payload` is not exported in a namespace, the only way I have is
```
type Payload = (Parameters<typeof Wreck.request>[2])['payload'];
```

Reply:
I think that's fine. I thought there was some way to do this, but I'm getting complains in my IDE:
```ts
import type {client} from '@hapi/wreck';
type Payload = typeof client.request.payload
```

Reply:
Thanks! It is impossible to import the type within a namespace :(